src/cheri/insns/jalr_32bit.adoc

```diff
@@ -14,22 +14,35 @@ include::wavedrom/ct-unconditional-2.adoc[]
 
 include::base_isa_extension.adoc[]
 
+#Begin changed since last ARC review#
+
 Description::
 Indirect jump to the target capability in `{cs1}` with an address offset.
 +
 Copy `{cs1}` to the target <<pcc>>
 +
-. If the target <<pcc>> is a <<sentry_cap>> and the `offset` is zero, then unseal the target <<pcc>>.
-Otherwise, increment the address of the target <<pcc>> by the sign-extended 12-bit `offset`.
+. If the platform includes the <<section_rvy_sentry_jalr_ext>> extension,
+  then impose any conditional behavior specified and,
+  if the target <<pcc>> is a sentry (that is, sealed with an acceptable type),
+  ensure that the `offset` immediate is zero and unseal the target <<pcc>>.
+. Increment the address of the target <<pcc>> by the sign-extended 12-bit `offset`.
 . Set bit zero of the target address to zero and update <<pcc>> with the new target address using the semantics of the <<SCADDR>> instruction
+. If `rd` is not `null`, add the width of this instruction
```

Comment:
Unrelated to the text but I've generally been using 1 line per sentence and relying on editor wrapping since that's recommended by asciidoctor: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line. This ensures that editing one sentence doesn't affect multiple lines.

Reply:
By the way, no need to make this change. this was just an observation.